src/pages/ToDoApp.tsx

```diff
@@ -54,47 +55,89 @@ export const ToDoApp = () => {
     saveTaskToLocalStorage(updatedTasks);
   };
 
+  const Container = styled.div`
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    min-height: 100vh;
+    background: #68d391;
+  `;
+  const CustomForm = styled.form`
+    width: 80%;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    flex-direction: column;
+    gap: 1rem;
+  `;
+
+  const UlCustom = styled.ul`
+    width: 52%;
+    display: flex-start;
+    align-items: center;
+    justify-content: center;
+    flex-direction: column;
+    background-color: #FFF59D;
+    border: 3px solid #333; 
+    border-radius: 10px;
+    box-shadow: 5px 10px 0px 5px #333;
+    padding: 10px 0;
+    position: relative;
+
+    h1 {
+    display: flex;
```

Comment:
create a style for h1 => const Heading1 = styled.h1`...`

Reply:
I think it as child of UlCustom so i wrote like that
Can you give me reasons why i should generate new variable to style that element,
Because i think it will be longer

Reply:
It should be consistent with our code conventions. UlCustom is a child of CustomForm, CustomForm is a child of Container.Why do we split it? We do this structure for better readability and easier adjustments as the file grows larger. With the same logics, it makes sense to generate a new variable for styling h1 and span